foladipo/docs-cabinet-cp2
# Chore/148471211/implement defence feedback

#### What does this PR do?

This PR implements a lot of feedback received during my CP2 defence and peer reviews. For the full list of feedback and how many this PR implements, see [this link](https://www.pivotaltracker.com/story/show/148471211).

#### Description of task to be completed

Implement feedback received during peer reviews and checkpoint defense.

#### How should this be manually tested?

- Follow the instructions stated in this project's README to set up your local instance of this app.

- Consequently, you should have sign up for a new account.

- Visit [this link](https://www.pivotaltracker.com/story/show/148471211) for a full list of the feedback implemented in this PR and test them one by one.

#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker stories?

148471211

#### Screenshots
<img width="1280" alt="screen shot 2017-07-08 at 11 32 30 am" src="https://user-images.githubusercontent.com/26963575/27984566-325445ac-63d1-11e7-96c2-3877476c8189.png">
